config/locales/placements/en/models/hosting_environment.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  models:
+    hosting_environment:
+      placements:
+        name: beta
```

Comment:
Hmm, this feels like we're using this dictionary for config. Can I suggest an env var as we don't use `Settings`?

Reply:
The think with using env vars though is that you don't allow each service do define their hosting envname

Reply:
Sure. In that case, should this be a constant in the `HostingEnvironment` module instead?